reverse_proxy.go

```diff
@@ -97,6 +97,10 @@ var hopHeaders = []string{
 // NewSingleHostReverseProxy does not rewrite the Host header.
 // To rewrite Host headers, use ReverseProxy directly with a custom
 // director policy.
+//
+// Note: it uses the default global client.Client when you do not pass config.ClientOption.
+// When using pass config.ClientOption it initializes a local client.Client.
```

Comment:
using pass -> passing

Reply:
it initializes a -> it will initialize a

Reply:
client.Client -> client.Client instance

Reply:
done